openlibrary/plugins/openlibrary/js/editions-table/index.js

```diff
@@ -1,8 +1,13 @@
 import 'datatables.net-dt';
 import '../../../../../static/css/legacy-datatables.less';
 
+const DEFAULT_LENGTH = 3
+const KEY = 'resultsLength'
```

Comment:
Give a more specific name than `KEY` :)

Also, since these are global across the entire site in local storage, you usually want more specific keys to avoid name collisions.

```suggestion
const LS_RESULTS_LENGTH_KEY = 'editions-table.resultsLength';
```

Reply:
I've made the requested changes, and also added missing semicolons.